tests/test_routes.py

```diff
@@ -59,6 +71,15 @@ def test_index(self):
         resp = self.client.get("/")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
+    def test_delete_product(self):
+        """Test deleting a product"""
+        test_product = self._create_product_id(1)[0]
```

Comment:
If you'll always be picking the first product in the list through [0], would it be better for _create_product_id to return just 1 product instead of a list of products?

This isn't a big deal though and can be addressed in the next sprint

Reply:
Yeah make sense, thanks